### ISSUE TITLE
gi.overrides.Gst.NotInitialized: Please call Gst.init(argv) before using GStreamer

### ISSUE THREAD

**Issue by conihorse** [jonian/kickoff-player#4]
Getting this error on Arch, installed via AUR https://aur.archlinux.org/packages/kickoff-player-git/

** (kickoff-player:11366): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Sia8LdPvZE: Connection refused
Traceback (most recent call last):
  File "/usr/bin/kickoff-player", line 109, in <module>
    player = KickoffPlayer()
  File "/usr/bin/kickoff-player", line 56, in __init__
    self.player   = PlayerHandler(self)
  File "/opt/kickoff-player-git/handlers/player.py", line 43, in __init__
    self.playbin = GstBox(callback=self.update_status)
  File "/opt/kickoff-player-git/widgets/gstbox.py", line 19, in __init__
    if not Gst.is_initialized():
  File "/usr/lib/python3.6/site-packages/gi/overrides/Gst.py", line 507, in fake_method
    raise NotInitialized("Please call Gst.init(argv) before using GStreamer")
gi.overrides.Gst.NotInitialized: Please call Gst.init(argv) before using GStreamer

**Comment by jonian** [comment]
Hi, thank you for the report.

Do you have any of the packages below installed:

gst-plugins-base 
gst-plugins-good
gst-plugins-bad
gst-plugins-ugly

If not, can you try if installing them solves your problem?

Gstreamer is being initialized [here](https://github.com/jonian/kickoff-player/blob/02d069663b890cd83efb56c98f598e8087b3e3e1/widgets/gstbox.py#L19).

**Comment by conihorse** [comment]
I notice gst-plugins-bad isn't installed - will try installing.

➜  ~ pacaur -Ss gstreamer | grep installed
extra/clutter-gst 3.0.24-1 [installed]
extra/gst-libav 1.12.2-1 [installed]
extra/gst-plugins-base 1.12.2-1 [installed]
extra/gst-plugins-base-libs 1.12.2-1 [installed]
extra/gst-plugins-good 1.12.2-1 [installed]
extra/gst-plugins-ugly 1.12.2-1 [installed]
extra/gst-python 1.12.2-1 [installed]
extra/gstreamer 1.12.2-1 [installed]
extra/phonon-qt4-gstreamer 4.9.0-3 [installed]
extra/phonon-qt5-gstreamer 4.9.0-3 [installed]

**Comment by conihorse** [comment]
after installing gst-plugins-bad, still the same error

➜  ~ pacaur -Ss gstreamer | grep installed
extra/clutter-gst 3.0.24-1 [installed]
extra/gst-libav 1.12.2-1 [installed]
extra/gst-plugins-bad 1.12.2-3 [installed]
extra/gst-plugins-base 1.12.2-1 [installed]
extra/gst-plugins-base-libs 1.12.2-1 [installed]
extra/gst-plugins-good 1.12.2-1 [installed]
extra/gst-plugins-ugly 1.12.2-1 [installed]
extra/gst-python 1.12.2-1 [installed]
extra/gstreamer 1.12.2-1 [installed]
extra/phonon-qt4-gstreamer 4.9.0-3 [installed]
extra/phonon-qt5-gstreamer 4.9.0-3 [installed]
---
➜  ~ kickoff-player             

(kickoff-player:15835): Gtk-WARNING **: Theme parsing error: gtk.css:68:35: The style property GtkButton:child-displacement-x is deprecated and shouldn't be used anymore. It will be removed in a future version

(kickoff-player:15835): Gtk-WARNING **: Theme parsing error: gtk.css:69:35: The style property GtkButton:child-displacement-y is deprecated and shouldn't be used anymore. It will be removed in a future version

(kickoff-player:15835): Gtk-WARNING **: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version

** (kickoff-player:15835): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Sia8LdPvZE: Connection refused
Traceback (most recent call last):
  File "/usr/bin/kickoff-player", line 109, in <module>
    player = KickoffPlayer()
  File "/usr/bin/kickoff-player", line 56, in __init__
    self.player   = PlayerHandler(self)
  File "/opt/kickoff-player-git/handlers/player.py", line 43, in __init__
    self.playbin = GstBox(callback=self.update_status)
  File "/opt/kickoff-player-git/widgets/gstbox.py", line 19, in __init__
    if not Gst.is_initialized():
  File "/usr/lib/python3.6/site-packages/gi/overrides/Gst.py", line 507, in fake_method
    raise NotInitialized("Please call Gst.init(argv) before using GStreamer")
gi.overrides.Gst.NotInitialized: Please call Gst.init(argv) before using GStreamer

**Comment by jonian** [comment]
The problem was caused from gst-python package. It is fixed now, but if you do not need the gst-python package, you should remove it since gstreamer works with python through python-gobject.

I have also updated the AUR package.

**Comment by conihorse** [comment]
confirmed fixed on my environment with latest AUR.

Thank you for your work on this, it looks excellent (potentially, just waiting for something to watch!).

**Comment by jonian** [comment]
If you want to try some channels, many of the Arenavision are live most of the time. Also some russian channels like Match Football 1.
